--- cdrdao-1.1.7/configure.ac.orig	Sun Oct  6 20:53:55 2002
+++ cdrdao-1.1.7/configure.ac	Thu Mar 13 21:06:12 2003
@@ -65,12 +65,12 @@
 AC_CHECK_FUNC(sched_setscheduler,AC_DEFINE(HAVE_SCHED_SETSCHEDULER),
   AC_CHECK_LIB(posix4,sched_setscheduler,[use_libposix4=yes;AC_DEFINE(HAVE_SCHED_SETSCHEDULER)]))
 
-AC_CHECK_LIB(socket,socket,[use_libsocket=yes])
-AC_CHECK_LIB(socket,connect,[use_libsocket=yes])
+AC_CHECK_FUNC(socket, , [AC_CHECK_LIB(socket,socket,[use_libsocket=yes])])
+AC_CHECK_FUNC(connect, , [AC_CHECK_LIB(socket,connect,[use_libsocket=yes])])
 
-AC_CHECK_LIB(nsl,gethostbyname,[use_libnsl=yes])
-AC_CHECK_LIB(nsl,inet_addr,[use_libnsl=yes])
-AC_CHECK_LIB(nsl,inet_ntoa,[use_libnsl=yes])
+AC_CHECK_FUNC(gethostbyname, , [AC_CHECK_LIB(nsl,gethostbyname,[use_libnsl=yes])])
+AC_CHECK_FUNC(inet_addr, , [AC_CHECK_LIB(nsl,inet_addr,[use_libnsl=yes])])
+AC_CHECK_FUNC(inet_ntoa, , [AC_CHECK_LIB(nsl,inet_ntoa,[use_libnsl=yes])])
 
 dnl Add 'libsocket' if required
 if test "$use_libsocket" = yes; then
