--- configure.in~	2004-07-10 13:22:45.144089480 +0200
+++ configure.in	2004-07-10 13:23:19.274900808 +0200
@@ -216,9 +216,9 @@
 fi
 if test "$xdao" = yes; then
   en_xdao=yes
-  PKG_CHECK_MODULES(GTKMM2, gtkmm-2.0 >= 2.2,[],
+  PKG_CHECK_MODULES(GTKMM2, gtkmm-2.4 >= 2.2,[],
         [echo "Building of gcdmaster disabled"; en_xdao=no])
-  PKG_CHECK_MODULES(LIBGUIMM2, libgnomeuimm-2.0 >= 2.0,[],
+  PKG_CHECK_MODULES(LIBGUIMM2, libgnomeuimm-2.6 >= 2.0,[],
         [echo "Building of gcdmaster disabled"; en_xdao=no])
 fi
 
