diff -Nur cdrdao-1.1.5.orig/dao/Makefile.in cdrdao-1.1.5/dao/Makefile.in
--- cdrdao-1.1.5.orig/dao/Makefile.in	Sun Apr 29 15:42:08 2001
+++ cdrdao-1.1.5/dao/Makefile.in	Fri Nov  2 12:59:50 2001
@@ -67,14 +67,14 @@
 all: $(PRG)
 
 install: all installdirs
-	$(INSTALL_PROGRAM) $(PRG) $(bindir)
-	$(INSTALL_DATA) cdrdao.man $(man1dir)/cdrdao.1
-	$(INSTALL_DATA)	cdrdao.drivers $(DRIVER_TABLE_FILE)
+	$(INSTALL_PROGRAM) $(PRG) $(DESTDIR)$(bindir)
+	$(INSTALL_DATA) cdrdao.man $(DESTDIR)$(man1dir)/cdrdao.1
+	$(INSTALL_DATA)	cdrdao.drivers $(DESTDIR)$(DRIVER_TABLE_FILE)
 
 installdirs:
-	$(srcdir)/../mkinstalldirs $(bindir)
-	$(srcdir)/../mkinstalldirs $(man1dir)
-	$(srcdir)/../mkinstalldirs $(CDRDAO_DATA_DIR)
+	$(srcdir)/../mkinstalldirs $(DESTDIR)$(bindir)
+	$(srcdir)/../mkinstalldirs $(DESTDIR)$(man1dir)
+	$(srcdir)/../mkinstalldirs $(DESTDIR)$(CDRDAO_DATA_DIR)
 
 clean:
 	rm -f *.o $(PRG)
diff -Nur cdrdao-1.1.5.orig/xdao/Makefile.in cdrdao-1.1.5/xdao/Makefile.in
--- cdrdao-1.1.5.orig/xdao/Makefile.in	Sun Apr 29 15:42:09 2001
+++ cdrdao-1.1.5/xdao/Makefile.in	Fri Nov  2 13:00:36 2001
@@ -70,23 +70,23 @@
 all: $(PRG)
 
 install: all installdirs
-	$(INSTALL_PROGRAM) $(PRG) $(bindir)
-	$(INSTALL_DATA) xcdrdao.man $(man1dir)/xcdrdao.1
-	$(INSTALL_DATA) gcdmaster.png $(PIXMAPS_DIR)/gcdmaster.png
-	$(INSTALL_DATA) pixmap_audiocd.png $(PIXMAPS_DIR)/pixmap_audiocd.png
-	$(INSTALL_DATA) pixmap_copycd.png $(PIXMAPS_DIR)/pixmap_copycd.png
-	$(INSTALL_DATA) pixmap_datacd.png $(PIXMAPS_DIR)/pixmap_datacd.png
-	$(INSTALL_DATA) pixmap_dumpcd.png $(PIXMAPS_DIR)/pixmap_dumpcd.png
-	$(INSTALL_DATA) pixmap_help.png $(PIXMAPS_DIR)/pixmap_help.png
-	$(INSTALL_DATA) pixmap_mixedcd.png $(PIXMAPS_DIR)/pixmap_mixedcd.png
-	$(INSTALL_DATA) pixmap_open.png $(PIXMAPS_DIR)/pixmap_open.png
-	$(INSTALL_DATA) gcdmaster.desktop $(datadir)/gnome/apps/Applications/gcdmaster.desktop
+	$(INSTALL_PROGRAM) $(PRG) $(DESTDIR)$(bindir)
+	$(INSTALL_DATA) xcdrdao.man $(DESTDIR)$(man1dir)/xcdrdao.1
+	$(INSTALL_DATA) gcdmaster.png $(DESTDIR)$(PIXMAPS_DIR)/gcdmaster.png
+	$(INSTALL_DATA) pixmap_audiocd.png $(DESTDIR)$(PIXMAPS_DIR)/pixmap_audiocd.png
+	$(INSTALL_DATA) pixmap_copycd.png $(DESTDIR)$(PIXMAPS_DIR)/pixmap_copycd.png
+	$(INSTALL_DATA) pixmap_datacd.png $(DESTDIR)$(PIXMAPS_DIR)/pixmap_datacd.png
+	$(INSTALL_DATA) pixmap_dumpcd.png $(DESTDIR)$(PIXMAPS_DIR)/pixmap_dumpcd.png
+	$(INSTALL_DATA) pixmap_help.png $(DESTDIR)$(PIXMAPS_DIR)/pixmap_help.png
+	$(INSTALL_DATA) pixmap_mixedcd.png $(DESTDIR)$(PIXMAPS_DIR)/pixmap_mixedcd.png
+	$(INSTALL_DATA) pixmap_open.png $(DESTDIR)$(PIXMAPS_DIR)/pixmap_open.png
+	$(INSTALL_DATA) gcdmaster.desktop $(DESTDIR)$(datadir)/gnome/apps/Applications/gcdmaster.desktop
 
 installdirs:
-	$(srcdir)/../mkinstalldirs $(bindir)
-	$(srcdir)/../mkinstalldirs $(man1dir)
-	$(srcdir)/../mkinstalldirs $(PIXMAPS_DIR)
-	$(srcdir)/../mkinstalldirs $(datadir)/gnome/apps/Applications
+	$(srcdir)/../mkinstalldirs $(DESTDIR)$(bindir)
+	$(srcdir)/../mkinstalldirs $(DESTDIR)$(man1dir)
+	$(srcdir)/../mkinstalldirs $(DESTDIR)$(PIXMAPS_DIR)
+	$(srcdir)/../mkinstalldirs $(DESTDIR)$(datadir)/gnome/apps/Applications
 
 uninstall:
 	rm -f $(bindir)/$(PRG) $(man1dir)/xcdrdao.1
