--- cdrdao-1.2.6/configure.ac.orig	2025-12-05 11:23:58.000000000 +0100
+++ cdrdao-1.2.6/configure.ac	2026-03-09 19:40:00.000000000 +0100
@@ -90,8 +90,8 @@
 AC_CHECK_FUNCS(sched_get_priority_max)
 AC_CHECK_FUNCS(sched_setscheduler)
 
-AC_CHECK_LIB(socket,socket,[use_libsocket=yes])
-AC_CHECK_LIB(socket,connect,[use_libsocket=yes])
+AC_CHECK_FUNC(socket, , [AC_CHECK_LIB(socket,socket,[use_libsocket=yes])])
+AC_CHECK_FUNC(connect, , [AC_CHECK_LIB(socket,connect,[use_libsocket=yes])])
 
 dnl Add 'libsocket' if required
 if test "$use_libsocket" = yes; then
