diff -Nur cdrdao-1.1.3.orig/paranoia/configure.in cdrdao-1.1.3/paranoia/configure.in
--- cdrdao-1.1.3.orig/paranoia/configure.in	Tue Oct 26 18:13:45 1999
+++ cdrdao-1.1.3/paranoia/configure.in	Thu Mar 15 01:42:51 2001
@@ -16,10 +16,10 @@
 	case $host in 
 	i?86-*-linux*)
 		DEBUG="-g -Wall -fsigned-char"
-		OPT="-O20 -ffast-math -fsigned-char -finline-functions";;
+		OPT="$CFLAGS -ffast-math -fsigned-char -finline-functions";;
 	*)
 		DEBUG="-g -Wall -fsigned-char"
-		OPT="-O20 -fsigned-char";;
+		OPT="$CFLAGS -fsigned-char";;
         esac
 fi
 
