--- cdrdao-1.2.6/configure.ac.orig	2025-12-05 11:23:58.000000000 +0100
+++ cdrdao-1.2.6/configure.ac	2026-03-09 19:40:00.000000000 +0100
@@ -183,8 +183,8 @@
 
   en_pccts="yes"
 else
-  AC_PATH_PROG(antlr_path,antlr,no,$pcctsbin)
-  AC_PATH_PROG(dlg_path,dlg,no,$pcctsbin)
+  AC_PATH_PROG(antlr_path,pccts-antlr,no,$pcctsbin)
+  AC_PATH_PROG(dlg_path,pccts-dlg,no,$pcctsbin)
 
   if test $antlr_path = "no" || test $dlg_path = "no"; then
     AC_MSG_WARN([PCCTS: antlr or dlg not found in path, using own.])
