diff -Nur cdrdao-1.2.0.org/trackdb/CueParser.cc cdrdao-1.2.0/trackdb/CueParser.cc
--- cdrdao-1.2.0.org/trackdb/CueParser.cc	2005-05-09 21:21:28.000000000 +0200
+++ cdrdao-1.2.0/trackdb/CueParser.cc	2005-09-17 15:46:15.638476312 +0200
@@ -23,6 +23,8 @@
 
 #include "Cue2Toc.h"
 
+class Toc;
+
 extern Toc *parseToc(const char* tocBuffer, const char *filename);
 
 Toc *parseCue(FILE *fp, const char *filename)
diff -Nur cdrdao-1.2.0.org/trackdb/FormatConverter.h cdrdao-1.2.0/trackdb/FormatConverter.h
--- cdrdao-1.2.0.org/trackdb/FormatConverter.h	2005-04-22 04:01:46.000000000 +0200
+++ cdrdao-1.2.0/trackdb/FormatConverter.h	2005-09-17 15:45:27.509792984 +0200
@@ -27,6 +27,8 @@
 #include "TrackData.h"
 #include "Toc.h"
 
+class Toc;
+
 // Quick abstract class declarations. Format converters should derive
 // their own FormatSupport and FormatSupportManager.
 
