--- cdrdao-1.2.1/trackdb/CueParser.cc.orig	2005-09-24 20:41:49.000000000 +0200
+++ cdrdao-1.2.1/trackdb/CueParser.cc	2005-11-26 02:35:15.000000000 +0100
@@ -24,6 +24,8 @@
 #include "Cue2Toc.h"
 #include "Toc.h"
 
+class Toc;
+
 extern Toc *parseToc(const char* , const char *);
 
 Toc *parseCue(FILE *fp, const char *filename)
diff -Nur cdrdao-1.2.0.org/trackdb/FormatConverter.h cdrdao-1.2.0/trackdb/FormatConverter.h
--- cdrdao-1.2.0.org/trackdb/FormatConverter.h	2005-04-22 04:01:46.000000000 +0200
+++ cdrdao-1.2.0/trackdb/FormatConverter.h	2005-09-17 15:45:27.509792984 +0200
@@ -27,6 +27,8 @@
 #include "TrackData.h"
 #include "Toc.h"
 
+class Toc;
+
 // Quick abstract class declarations. Format converters should derive
 // their own FormatSupport and FormatSupportManager.
 
#!/bin/sh /usr/share/dpatch/dpatch-run
## 09-gcc-3.4.dpatch by Matthias Klose <doko@debian.org>
##
## DP: Fix build failures with g++-4.3 (Closes: #455309).

@DPATCH@

diff -Naurp cdrdao-1.2.2.orig/trackdb/FormatMp3.cc cdrdao-1.2.2/trackdb/FormatMp3.cc
--- cdrdao-1.2.2.orig/trackdb/FormatMp3.cc	2005-09-24 20:28:43.000000000 +0000
+++ cdrdao-1.2.2/trackdb/FormatMp3.cc		2008-03-07 11.14.07.000000000 +0000
@@ -25,6 +25,7 @@
 #include <unistd.h>
 #include <string.h>
 #include <fcntl.h>
+#include <cstring> 
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
--- cdrdao-1.2.3/dao/ScsiIf-linux.cc.orig	2009-09-28 12:12:03.000000000 +0200
+++ cdrdao-1.2.3/dao/ScsiIf-linux.cc	2010-06-03 17:47:13.502076082 +0200
@@ -26,6 +26,8 @@
 #include <string.h>
 #include <assert.h>
 #include <sys/ioctl.h>
+#include <sys/types.h>
+#include <sys/stat.h>
 #include <glob.h>
 #include <asm/param.h>
 #include <scsi/scsi.h>
