--- cdrdao-1.2.1/trackdb/CueParser.cc.orig	2005-09-24 20:41:49.000000000 +0200
+++ cdrdao-1.2.1/trackdb/CueParser.cc	2005-11-26 02:35:15.000000000 +0100
@@ -24,6 +24,8 @@
 #include "Cue2Toc.h"
 #include "Toc.h"
 
+class Toc;
+
 extern Toc *parseToc(const char* , const char *);
 
 Toc *parseCue(FILE *fp, const char *filename)
diff -Nur cdrdao-1.2.0.org/trackdb/FormatConverter.h cdrdao-1.2.0/trackdb/FormatConverter.h
--- cdrdao-1.2.0.org/trackdb/FormatConverter.h	2005-04-22 04:01:46.000000000 +0200
+++ cdrdao-1.2.0/trackdb/FormatConverter.h	2005-09-17 15:45:27.509792984 +0200
@@ -27,6 +27,8 @@
 #include "TrackData.h"
 #include "Toc.h"
 
+class Toc;
+
 // Quick abstract class declarations. Format converters should derive
 // their own FormatSupport and FormatSupportManager.
 
