diff -urN cdrdao-1.1.9.org/dao/Makefile.am cdrdao-1.1.9/dao/Makefile.am
--- cdrdao-1.1.9.org/dao/Makefile.am	2004-06-13 22:08:01.981639208 +0200
+++ cdrdao-1.1.9/dao/Makefile.am	2004-06-13 22:08:52.371978712 +0200
@@ -96,7 +96,7 @@
 DRIVER_TABLE_FILE = $(CDRDAO_DATA_DIR)/drivers
 
 install-data-hook:
-	mv $(CDRDAO_DATA_DIR)/cdrdao.drivers $(DRIVER_TABLE_FILE)
+	mv -f $(DESTDIR)$(CDRDAO_DATA_DIR)/cdrdao.drivers $(DESTDIR)$(DRIVER_TABLE_FILE)
 
 INCLUDES =						\
 	-I$(srcdir)/../trackdb				\
