--- cdrdao-1.1.7/xdao/Makefile.in.orig	Wed Dec 25 21:09:53 2002
+++ cdrdao-1.1.7/xdao/Makefile.in	Wed Dec 25 21:12:03 2002
@@ -72,7 +72,7 @@
 
 install: all installdirs
 	$(INSTALL_PROGRAM) $(PRG) $(DESTDIR)$(bindir)
-	$(INSTALL_DATA) xcdrdao.man $(DESTDIR)$(man1dir)/xcdrdao.1
+	$(INSTALL_DATA) xcdrdao.man $(DESTDIR)$(man1dir)/gcdmaster.1x
 	$(INSTALL_DATA) gcdmaster.png $(DESTDIR)$(PIXMAPS_DIR)/gcdmaster.png
 	$(INSTALL_DATA) pixmap_audiocd.png $(DESTDIR)$(PIXMAPS_DIR)/pixmap_audiocd.png
 	$(INSTALL_DATA) pixmap_copycd.png $(DESTDIR)$(PIXMAPS_DIR)/pixmap_copycd.png
@@ -90,13 +90,13 @@
 	$(INSTALL_DATA) pixmap_zoom-out.xpm $(DESTDIR)$(PIXMAPS_DIR)/pixmap_zoom-out.xpm
 	$(INSTALL_DATA) pixmap_zoom-selection.xpm $(DESTDIR)$(PIXMAPS_DIR)/pixmap_zoom-selection.xpm
 	$(INSTALL_DATA) pixmap_zoom-fit.xpm $(DESTDIR)$(PIXMAPS_DIR)/pixmap_zoom-fit.xpm
-	$(INSTALL_DATA) gcdmaster.desktop $(DESTDIR)$(datadir)/gnome/apps/Applications/gcdmaster.desktop
+	$(INSTALL_DATA) gcdmaster.desktop $(DESTDIR)$(datadir)/applnk/Utilities/CD-RW/gcdmaster.desktop
 
 installdirs:
 	$(srcdir)/../mkinstalldirs $(DESTDIR)$(bindir)
 	$(srcdir)/../mkinstalldirs $(DESTDIR)$(man1dir)
 	$(srcdir)/../mkinstalldirs $(DESTDIR)$(PIXMAPS_DIR)
-	$(srcdir)/../mkinstalldirs $(DESTDIR)$(datadir)/gnome/apps/Applications
+	$(srcdir)/../mkinstalldirs $(DESTDIR)$(datadir)/applnk/Utilities/CD-RW
 
 uninstall:
 	rm -f $(bindir)/$(PRG) $(man1dir)/xcdrdao.1
